Move coding conventions to wiki

hctrdev opened the pull request:
## Description

Remove the coding conventions section from the Contributing file and move them to the project wiki.

Commit:
#33: Move coding conventions to wiki

Remove the coding conventions section from the Contributing file and move them to the project wiki.

ColmBhandal commented on CONTRIBUTING.md:
```diff
@@ -40,12 +40,4 @@
-| Variable Naming | As far as possible, variables should be named so that the meaning of the variable can be inferred from the variable. name. You can always start coding with variables like `i` and `j` and then when it's time to commit your code, you can use your IDE to rename the variables to something more descriptive e.g. `rowIndex` and `columnIndex`.|
-
-...More conventions will follow. Please be patient with us as we convert our coding experience to text...
+You can find the latest coding conventions in the [wiki](https://github.com/ColmBhandal/CsharpExtras/wiki/Coding-Conventions).
```
I would say something stronger here. E.g.

As part of our contributing guidelines, we recommend you check out our coding conventions. This will help streamline our process as we will be less likely to require minor changes on pull requests. Please check out the changes on our Wiki if this is your first time contributing to the repo: [LINK]

ColmBhandal reviewed (changes requested):
See my comment

Commit:
#33: Reword coding conventions section

Update the Contributing file with stronger wording around where to find the coding conventions.
